functorch/csrc/dim/dim_opcode.c

```diff
@@ -0,0 +1,6 @@
+#include <torch/csrc/utils/python_compat.h>
+#if defined(_WIN32) && IS_PYTHON_3_11_PLUS
+#define Py_BUILD_CORE
+#define NEED_OPCODE_TABLES
+#include "internal/pycore_opcode.h"
```

Comment:
We already include these in https://github.com/pytorch/pytorch/blob/08657b82f51b183f01a37de30ec45c34392acd28/torch/csrc/dynamo/cpython_defs.c#L30 not sure why that is not enough?

Reply:
Well that file is compiled in torch_python which is a dll. They would need to be marked as dll_export /dll_import in order to access them in functorch.

Reply:
Ho these are still different libs. Ok.